docs/BreakingChanges/077 DCOM and reflection.md

```diff
@@ -0,0 +1,30 @@
+## 77: DCOM and reflection
```

Comment:
{
  "Id": "77",
  "Title": "DCOM and reflection",
  "Details": "Reflection objects can no longer be passed from managed code to out-of-process DCOM clients. The following types are affected:\n\n- Assembly\n- MemberInfo (and its derived types, including FieldInfo, MethodInfo, Type, and TypeInfo)\n- MethodBody\n- Module\n- ParameterInfo.\n\nCalls to `IMarshal` for the object return `E_NOINTERFACE`.",
  "Suggestion": "Update marshaling code to work with non-reflection objects",
  "Link": "https://msdn.microsoft.com/en-us/library/dn833125(v=vs.110).aspx#Core",
  "Markdown": null,
  "ApplicableApis": [
    "TBD: Can we detect this?"
  ],
  "Related": null,
  "VersionBroken": "4.6",
  "VersionFixed": null,
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": null,
  "Notes": null,
  "IsRetargeting": false,
  "ImpactScope": 2
}

**Parsed APIs:**
- [ ] TBD: Can we detect this?